chart/templates/gratia-output-config.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
This whole file should be wrapped in "if .Values.outputFormat = gratia" don't you think?

I suppose the same is true for the ssmsend-config and ssmsend-secret templates, if the format is ssmsend.

Reply:
That makes sense, I've updated all 3 files.